style/style.css

```diff
@@ -103,3 +117,159 @@ nav {
 .button-mobile {
   margin-right: 15px;
 }
+
+.work-inside {
+  display: grid;
+  grid-template-rows: repeat(3, auto);
+  list-style: none;
+  padding: 8rem 1.6rem;
+  row-gap: 5em;
+  justify-content: center;
+}
+
+.project {
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 20px;
+  background-color: #fff;
+}
```

Comment:
- [ ] Very nice job so far 🥇 however according to the requirements the elements in every individual project should be aligned using flexbox